homeassistant/components/device_tracker/__init__.py

```diff
@@ -375,6 +426,9 @@ def async_setup_group(self):
                     ATTR_NAME: GROUP_NAME_ALL_DEVICES,
                     ATTR_ENTITIES: entity_ids}))
 
+    # Think about what to do to support update of stale device
+    # and consider home.
+
```

Comment:
Do we even need to continue tracking stale devices?  Why not just drop them out of tracking automatically?

Reply:
The problem is to know if they are stale or not, I think. That's what the existing logic is trying to tell.

Reply:
I don't know the devicetracker well enough, but why do we care if it's stale?

Reply:
Devices that are scanned by a router via wifi can go into power save and then the router will not pick them up anymore, even if they are still at home. To counter this there's a time period that is configurable where the device tracker still considers the device as home even if it's not seen.